src/browser/components/SettingsPage.jsx

```diff
@@ -65,99 +47,124 @@ export default class SettingsPage extends React.Component {
     this.handleShowUnreadBadge = this.handleShowUnreadBadge.bind(this);
     this.handleChangeUseSpellChecker = this.handleChangeUseSpellChecker.bind(this);
     this.handleChangeEnableHardwareAcceleration = this.handleChangeEnableHardwareAcceleration.bind(this);
+    this.saveSetting = this.saveSetting.bind(this);
+    this.updateSaveState = this.updateSaveState.bind(this);
```

Comment:
Nothing to do now, but we should remove all this function bindings in favor of arrow functions to simplify the code

Reply:
Converted in this component. Quick regex. :)